R/stat-smooth.r

```diff
@@ -127,12 +124,25 @@ StatSmooth <- ggproto("StatSmooth", Stat,
         xseq <- seq(range[1], range[2], length.out = n)
       }
     }
-    # Special case span because it's the most commonly used model argument
+
+    ## Special case span because it's the most commonly used model argument
     if (identical(method, "loess")) {
       method.args$span <- span
     }
 
-    if (is.character(method)) method <- match.fun(method)
+    ## If gam and gam's method is not specified by the user then use REML
+    if (identical(method, "gam") & is.null(method.args$method)) {
+        method.args$method <- "REML"
+    }
+
+    if (is.character(method)) {
+        if (identical(method, "gam")) {
```

Comment:
Can you please check the style here? You can use the styler package to match our usual guidelines (https://style.tidyverse.org)

Reply:
Should be OK now; used styler on stat-smooth.R